eventdata/operations/indexing.json

```diff
@@ -115,6 +115,25 @@
   },
   "index_template_name": "elasticlogs_i_write"
 },
+{
+  "name": "create_elasticlogs_index",
+  "operation-type": "createindex",
```

Comment:
I'm not a 100% sure but I wonder whether you can use the core Rally operation [`create-index`](http://esrally.readthedocs.io/en/stable/track.html#create-index) instead.

Reply:
That may be possible, but I will leave it as is for now.

Reply:
Makes sense.